packages/plugin-meye-config/css/main.css

```diff
@@ -0,0 +1,230 @@
+body {
```

Comment:
just as a note, it's usually better to just keep the CSS as <style> tags to be injected into the display element, either that or you could alternatively use the `survey` plugin way of using sass to also export a css file that can be implemented with the script tags or whatnot